### ISSUE TITLE
playlist.json has duplicate info

### ISSUE THREAD

**Issue by Rexadev** [yogeshwaran01/spotify-playlist-to-youtube-playlist#8]
"spotify_playlist_id": "xx",
same as
    "spotify_link": "https://open.spotify.com/playlist/xxx",

    "youtube_playlist_id": "xx",
same as
    "youtube_link": "https://www.youtube.com/playlist?list=xx"

**Comment by yogeshwaran01** [comment]
No, duplicate info. `spotify_playlist_id` is id of the playlist and `spotify_link` is the link to open in browser. Same for youtube

**Comment by Rexadev** [comment]
@yogeshwaran01 can't you get 1 from other ?

**Comment by yogeshwaran01** [comment]
Yes, It can be get from other. Just for easy to open in browser or copy.

**Comment by Rexadev** [comment]
> Yes, It can be get from other. Just for easy to open in browser or copy.

So just have the URL versions
